views/newEvent.handlebars

```diff
@@ -0,0 +1,28 @@
+    <section class="p-2">
+        <div class="card text-center">
+            <div class="card-header">
+              Create an event
+            </div>
+            <div class="card-body">
+                <form>
+                    <div class="form-group p-2">
```

Comment:
`name` and `id` attributes needed to be added/updated at their respective `<input>`: `name`, `address`, `description`, `date`. Currently, `id`s refer to the creation of a new user rather than a new event